docs/modules/ROOT/pages/how-to-guides/proc_managed_services_onboarding.adoc

```diff
@@ -3,101 +3,72 @@ When an application is ready for release, the Development team contacts the Mana
 
 The Managed Environment team creates the following objects on the Managed workspace:
 
-* *ReleaseStrategy* - The `ReleaseStrategy` object, created by the Managed Environment team, specifies the release pipeline that the system uses to release an application in their environment, as well as which enterprise contract the system must first validate.
-
-* *ReleasePlanAdmission* - The Managed Environment team creates the ReleasePlanAdmission object in response to the `ReleasePlan` object created by the development team. It indicates that the Managed Environment team has approved the xref:how-to-guides/proc_release_application.adoc#_creating_a_releaseplan_object[`ReleasePlan`] object for an application. It also specifies a `ReleaseStrategy` to be used for that particular Application and, optionally, an Environment to which the application will be deployed.
+* *ReleasePlanAdmission* - The Managed Environment team creates the `ReleasePlanAdmission` object in response to the `releasePlan` object created by the development team. It indicates that the Managed Environment team has approved the link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/proc_release_plan/[releasePlan] object for an application.
 
 * *`copy-application.sh`* - The Managed Environment team runs the `copy-application.sh` script by defining the application name before releasing an application to production. The script copies the application and all of its components from the Development to the Managed workspace. As a result, when releasing an application to production, the Managed Environment team uses the application on its workspace rather than the one on the Developer workspace.
 Additionally, if the Development team adds a new component to the application on their workspace,  they must notify the Managed Environment team. So they can run the `copy-application.sh` script again to ensure that the managed workspace has the most recent version of the application.
 
-== The `ReleaseStrategy` object
+== Creating a `ReleasePlanAdmission` object
 
 .*Prerequisites*
 
 * An existing Development and Managed workspace.
 
-* An existing `ReleasePlan` object.
+* An existing `ReleasePlan` object in the Development workspace.
 
 .*Procedures*
 
-* Create a release_strategy.yaml object by running:
+. Create a `ReleasePlanAdmission.yaml` object locally.
 
 +
-[source,]
-----
-oc apply -f releasestrategy.yaml -n managed
-----
-
-+ 
-*Example release_strategy.yaml object*
+*Example `ReleasePlanAdmission.yaml` object*
 
 +
 [source,yaml]
 ----
 apiVersion: appstudio.redhat.com/v1alpha1
-kind: ReleaseStrategy <.>
+kind: ReleasePlanAdmission <.>
 metadata:
-  name: strategy <.>
-  namespace: managed-environment <.>
+ name: sre-production <.>
+ namespace: managed <.>
```

Comment:
I think these docs would be a lot easier to follow if consistent values were used. For example, in the example ReleasePlan, the managed namespace is called `managed-workspace` (the target), yet here namespace is called `managed`